src/superio/nuvoton/common/early_serial.c

```diff
@@ -83,3 +83,32 @@ void nuvoton_enable_serial(pnp_devfn_t dev, u16 iobase)
 	pnp_set_enable(dev, 1);
 	nuvoton_pnp_exit_conf_state(dev);
 }
+
+void nuvoton_assign_io_port_gpio(pnp_devfn_t dev, u16 iobase)
```

Comment:
@piotr-kleins please remove, this is not needed

Reply:
@miczyg1 Explained above why it is needed.